### PR TITLE
CASMPET-6650 MTL-2032

### PR DESCRIPTION
**Pull request by rustydb**
# Description

Upload RPMs and container images to pit nexus.

This sets the stage for image changes that pull RPMs and images instead of baking them in.

This new script must be invoked after `pit-init`, specifically after the `customizations.yaml` file has been generated and modified by the user.

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: install/csm-install/README.md [hunk 1]
```diff
@@ -78,15 +80,23 @@ See [Pre-installation](../pre-installation.md).
 
 See [Boot installation environment](../pre-installation.md#1-boot-installation-environment).
 
-### 3. Import CSM tarball
+### 3. Download the CSM tarball
 
-See [Import CSM tarball](../pre-installation.md#2-import-csm-tarball).
+See [Import CSM tarball](../pre-installation.md#2-download-and-extract-the-csm-tarball).
```
Comment (leliasen-hpe) [inline install/csm-install/README.md:85]:
```suggestion
See [Download and extract the CSM tarball](../pre-installation.md#2-download-and-extract-the-csm-tarball).
```

File: install/csm-install/README.md [hunk 2]
```diff
@@ -78,15 +80,23 @@ See [Pre-installation](../pre-installation.md).
...
 
 ### 4. Create system configuration
 
 See [Create system configuration](../pre-installation.md#3-create-system-configuration).
 
-### 5. Configure management network switches
+### 5. Import the CSM tarball
+
+See [Create system configuration](../pre-installation.md#4-import-the-csm-tarball).
```
Comment (leliasen-hpe) [inline install/csm-install/README.md:93]:
```suggestion
See [Import the CSM tarball](../pre-installation.md#4-import-the-csm-tarball).
```

File: install/pre-installation.md [hunk 3]
```diff
@@ -9,16 +9,16 @@ The page walks a user through setting up the Cray LiveCD with the intention of i
     1. [Prepare the data partition](#14-prepare-the-data-partition)
     1. [Set reusable environment variables](#15-set-reusable-environment-variables)
     1. [Exit the console and log in with SSH](#16-exit-the-console-and-log-in-with-ssh)
-1. [Import CSM tarball](#2-import-csm-tarball)
-    1. [Download CSM tarball](#21-download-csm-tarball)
-    1. [Import tarball assets](#22-import-tarball-assets)
+1. [Import CSM tarball](#2-download-and-extract-the-csm-tarball)
```
Comment (leliasen-hpe) [inline install/pre-installation.md:12]:
```suggestion
1. [Download and extract the CSM tarball](#2-download-and-extract-the-csm-tarball)
```

File: install/pre-installation.md [hunk 4]
```diff
@@ -690,6 +666,40 @@ Follow the [Prepare Site Init](prepare_site_init.md) procedure.
...
+   > ***NOTE*** The `${releasever_major}` and `${releasever_minor}` variables are interpolated by Zypper, the URI
+   > is intentionally wrapped with single-quotes to prevent the shell from interpolating them. Zypper will replace
+   > these variables with the currently running distributions major and minor version numbers.
+
+   ```bash
+   zypper addrepo --no-gpgcheck --refresh http://packages/repository/csm-noos csm-noos
+   zypper addrepo --no-gpgcheck --refresh 'http://packages/repository/csm-sle-${releasever_major}sp${releasever_minor}' 'csm-sle'
+   ```
+
+1. (`pit#`) Install `csm-testing` RPM.
```
Comment (leliasen-hpe) [inline install/pre-installation.md:691]: Should command say "Install csm-testing, canu, craycli and iuf-cli" RPMs?

Comment (rustydb) [reply install/pre-installation.md:691]: It sure should.

Comment (rustydb) [reply install/pre-installation.md]: Fixed.

**Review by leliasen-hpe — APPROVED** [review]

**Review by nrockershousen — APPROVED** [review]

**Pull request merged**